<?xml version="1.0" encoding="utf-8"?>
<launch>
  <arg name="computer_name" value="usl" />
<!--   <param name = "/use_sim_time" value="true"/> -->
    <node  pkg="random_frame_generator" name="random_frame_generator_basler_os" type="random_frame_generator_node" output="screen">
         <param name="image_folder_name" value="$(find random_frame_generator)/data/basler_os/images"/>
         <param name="pointcloud_folder_name" value="$(find random_frame_generator)/data/basler_os/lidar_data"/>
         <param name="lidar_output_topic_name" value="/os1_cloud_node/points/rotated"/>
         <param name="image_output_topic_name" value="/pylon_camera_node/image_raw"/>
         <param name="output_fps" value="1"/>
         <param name="lidar_frame_id" value="os1_lidar"/>
         <param name="randomize_frames" value="false"/>
  </node>
</launch>
